Resources/Prototypes/Recipes/Construction/Graphs/furniture/meatspike.yml

```diff
@@ -3,6 +3,8 @@
   start: start
   graph:
     - node: start
+      actions:
+        - !type:DeleteEntity {}
```

Comment:
shouldnt this be part of the `to: start` edge completions

Reply:
I tested this with both construction and deconstruction both ways and it works fine both ways. Looking at other construction graphs, it's also done both ways so i genuinely can't say how it's supposed to be done.

Reply:
![image](https://user-images.githubusercontent.com/98561806/183255423-554dcffa-aa8d-4c0b-b013-bb0aaa014dfb.png)
official ruling.